### PR TITLE
Add Requirements

### PR DESCRIPTION
**Pull request by hide-hayashi**
I have written install environment specification. and I fixed  "support language" link  in  "getting started". Because it was noticed when I build mkdocs.

### CODE REVIEW

File: docs/installation_and_administration_manual/install.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Install Meteoroid
 
+## Install enviroment
+
+The following enviroment can run Meteoroid with FIWRE(orion) like [Getting Started](../getting_started.md). So you can easily start using Meteoroid in your machine.
```
Comment (YujiAzama) [inline docs/installation_and_administration_manual/install.md]: "Environment" is misspelled.

File: docs/installation_and_administration_manual/install.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Install Meteoroid
 
+## Install enviroment
```
Comment (YujiAzama) [inline docs/installation_and_administration_manual/install.md]: I think [Requirements] is a good title. 
And could you add Software Requirements? (e.g. Docker Version, Docker Compose Version)

File: docs/installation_and_administration_manual/install.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
 # Install Meteoroid
 
+## Requirements
+
+The following environment can run Meteoroid with FIWRE(orion) like [Getting Started](../getting_started.md). So you can easily start using Meteoroid in your machine.
```
Comment (ryo-ma) [inline docs/installation_and_administration_manual/install.md]: Typo **FIWRE**.

Comment (hide-hayashi) [reply docs/installation_and_administration_manual/install.md]: Thanks

**Review by ryo-ma — APPROVED** [review]
LGTM

**Pull request merged**